mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg

```diff
@@ -0,0 +1,7 @@
+include('templates/centos-stream-9.tpl')
+include('templates/epel-9.tpl')
+include('templates/epel-next-9.tpl')
```

Comment:
All 2 EPEL templates have a local repo. Does the latter win, or how does that work?

Reply:
That's... a good question. I'm not sure what happens here, but in any case, we need to dedupe these references. We have _three_ `local` references across these templates. And that's true for both 8 and 9.

I'm tempted to rename the EPEL ones to `epel-local` and `epel-next-local`. We'd need to document this change somewhere so people know how to use it, though.

Reply:
That's... a good question. I'm not sure what happens here, but in any case, we need to dedupe these references. We have _three_ `local` references across these templates. And that's true for both 8 and 9.

I'm tempted to rename the EPEL ones to `epel-local` and `epel-next-local`. We'd need to document this change somewhere so people know how to use it, though.

@carlwgeorge, what do you think?

Reply:
Perhaps rename them to `local-epel` and `local-epel-next`, so users can just do `--enablerepo 'local*'`.

Reply:
Oh, that makes sense! I'll make a separate PR for that.

Reply:
Done in #824